presto-main/src/main/java/io/prestosql/operator/TopNRankingOperator.java

```diff
@@ -42,9 +43,17 @@
     public static class TopNRankingOperatorFactory
             implements OperatorFactory
     {
+        public enum RankingType
```

Comment:
it's fine to use `RankingType` from `TopNRankingNode`. That should simplify imports